Vagrantfile

```diff
@@ -7,11 +7,11 @@ Vagrant.configure("2") do |config|
   # Online documentation: https://docs.vagrantup.com.
   # Boxes: https://vagrantcloud.com/search.
 
-  # ubuntu/focal64 box used instead of debian/stretch64 because
```

Comment:
I forgot to update my master and rebase the branch to master, so the revert to xenial didn't show up
It shows up now